.github/workflows/ci.yaml

```diff
@@ -259,35 +169,3 @@ jobs:
       - uses: codecov/codecov-action@v3
         with:
           files: lcov.info
-
-  notify:
-    runs-on: ubuntu-latest
-    if: github.event_name == 'push' && failure()
-    needs:
-      - lint
-      - deny
-      - sort
-      - clippy
-      - build
-      - test
-      - doc
-      - coverage
-    steps:
-      - name: Notify Discord on failure
```

Comment:
We've pulled back from doing dev/eng efforts in discord, so this only creates noise that doesn't help at this time.
While it would be nice to have a slack notification I don't think it's worth the effort at this time to implement